File: Timeconversion.js
function timeConversion(s) {
    var a = s.split(':')
    console.log(a)
    var b = a[2].slice(-2)
    console.log(b)
    if (parseInt(a[0]) < 12 && b =='PM'){
    	a[0] = parseInt(a[0])+12
		a[2] = a[2].replace(b,'')
    }
    else if(parseInt(a[0]) ==12 &&  b =="AM"){
		a[0] = parseInt(a[0]) - 12 + '0'
		a[2] = a[2].replace(b,'')
	}
	else {
		a[2] = a[2].replace(b,'')
	}

	return a.join(':')
}

console.log(timeConversion('07:05:45PM'))

File: Timeconversion.py
def timeConversion(s):
	a = s.split(':')
	b = s[-2:]
	print(b)
	if int(a[0]) < 12 and b =='PM':
		a[0] = str(int(a[0])+12)
		a[2] = a[2].replace(b,'')
	elif int(a[0]) ==12 and  b =="AM":
		a[0] = str(int(a[0]) - 12) + '0'
		a[2] = a[2].replace(b,'')
	else:
		a[2] = a[2].replace(b,'')

	return ":".join(a)

print(timeConversion('06:40:03AM')) 